_posts/2021-02-01-launching-the-solidity-forum.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+published: true
+title: 'Launching the Solidity Forum 🗃️'
+date: '2021-02-01'
+author: Franziska Heintel
+category: Announcements
+---
+
+In our effort to foster exchange of information, encourage more developers to give feedback about Solidity and join the discussions on language design and future direction of the compiler, we are happy to launch the [Solidity forum](https://forum.soliditylang.org/) today! 
+
+Moving forward the Solidity forum will be the dedicated place to discuss topics and questions related to…
+
+- The implementation and design of Solidity programming language.
```

Comment:
Also implementation?

Reply:
I don't understand? It says "implementation and design"

Reply:
Yes, but I'm asking if implementation discussion should not be done in github instead.

Reply:
Oooh now I get it. Yes, that's a fair point. Will remove it.